src/app/administrator/administrator.service.ts

```diff
@@ -28,6 +28,13 @@ export class AdministratorService {
       .then(response => response.json().data as Administrator)
       .catch(this.handleError);
   }
+  getAdministratorUsername(username: string): Promise<Administrator> {
```

Comment:
Is this being called anywhere in your changes?
If not it can be removed as it is being implemented in a forthcoming pull request for #68

Reply:
It is not